identity-apps-core/apps/authentication-portal/src/main/webapp/execution-flow.jsp

```diff
@@ -180,27 +184,40 @@
                 const [ error, setError ] = useState(null);
                 const [ postBody, setPostBody ] = useState(undefined);
                 const [ flowError, setFlowError ] = useState(undefined);
+                const [confirmationEffectDone, setConfirmationEffectDone] = useState(false);
 
                 useEffect(() => {
                     const savedFlowId = localStorage.getItem("flowId");
 
                     if (code !== "null" && state !== "null") {
                         setPostBody({
                             flowId: savedFlowId,
-                            flowType: "REGISTRATION",
```

Comment:
Is it okay to remove flow type here?

Reply:
Since we have the flowId, flow type is not needed. This is the redirection from google, at the moment we ar enot appending flow type in the params as well.

Reply:
Ack